### PR TITLE
added ccny

### LINKED ISSUE #6: City College needed in institutions

### PR DESCRIPTION
**Pull request by ahimelstein**
closed #6

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -205,6 +205,15 @@ ucopenhagen:
   name: University of Copenhagen
   state: ''
   zip: ''
+ccny:
+  aka: City College, CUNY, City College
```
Comment (sbillinge) [inline db/institutions.yml]: I think this should be a python list, so ``[City College, CUNY]`` and so on.

File: db/institutions.yml [hunk 2]
```diff
@@ -205,6 +205,15 @@ ucopenhagen:
...
+  city: New York
+  country: USA
+  departments: fixme
```
Comment (sbillinge) [inline db/institutions.yml]: just add a few departments, your favorite ones and add them here.

File: db/institutions.yml [hunk 3]
```diff
@@ -205,6 +205,16 @@ ucopenhagen:
   name: University of Copenhagen
   state: ''
   zip: ''
+ccny:
+  aka: [City College, CUNY City College]
+  city: New York
+  country: USA
+  departments: {cheme: Department of Chemical Engineering, chemistry: Department of Chemisty and Biochemistry}
+  name: The City College of New York
+  schools: Grove School of Engineering
```
Comment (sbillinge) [inline db/institutions.yml]: I think this should also be a dict, similar format to the departments.

Comment (ahimelstein) [reply db/institutions.yml]: for the aka or the department

Comment (sbillinge) [reply db/institutions.yml]: the schools.  Please look at Columbiau as an example.

File: db/institutions.yml [hunk 4]
```diff
@@ -211,7 +211,7 @@ ccny:
   country: USA
   departments: {cheme: Department of Chemical Engineering, chemistry: Department of Chemisty and Biochemistry}
   name: The City College of New York
-  schools: Grove School of Engineering
+  schools: {grove: The Grove School of Engineering, aka: [Grove School, Grove]}
```
Comment (sbillinge) [inline db/institutions.yml]: excellent

**Comment by sbillinge** [comment]
closes #6 
Nice job

**Pull request merged**